lib/languageFeatures/findDefinition.ts

```diff
@@ -14,10 +15,16 @@ export async function findDefinitions(linter: VhdlLinter, position: Position): P
     }
     return [];
   }))];
-  return candidateDefinitions.flatMap(definition => ({
-    targetRange: definition.range.copyExtendBeginningOfLine().getLimitedRange(10),
-    targetSelectionRange: definition.lexerToken?.range ?? definition.range.copyExtendBeginningOfLine().getLimitedRange(1),
-    text: definition.rootFile.originalText,
-    targetUri: URI.file(definition.rootFile.file).toString()
-  }));
+  return candidateDefinitions.flatMap(definition => {
+    const targetRange = (definition instanceof ORecordChild)
+    ? definition.parent.range.copyWithNewEnd(definition.range.end).getLimitedRange(5, true)
+    : definition.range.copyExtendBeginningOfLine().getLimitedRange(5);
+    const targetSelectionRange = definition.lexerToken?.range ?? definition.range.copyExtendBeginningOfLine().getLimitedRange(1);
+    return {
```

Comment:
I do not really like this, now it randomly gets extended to the beginning but not to the end...
But we maybe make this a seperate issue...

Reply:
For big records it is infeasible to show the full record because it doesn't fit into one popup and it is not possible to see the actual definition. By expanding it to the top I wanted to include the record name if the record is small enough.
If you have a good suggestion, I am happy to see it. But yes, maybe extract an issue from this conversation.